wideq/client.py

```diff
@@ -169,10 +170,10 @@ def load(cls, state):
 
         return client
 
-    def dump(self):
+    def dump(self) -> Dict[str, Any]:
         """Serialize the client state."""
 
-        out = {
+        out: Dict[str, Any] = {
```

Comment:
Just checking—is this type annotation necessary? It doesn't seem to make things any clearer, and I would have guessed that MyPy would have been able to infer it. But I imagine I'm probably wrong, which is why you added it. :smiley:

Reply:
I can check again, but sometimes mypy is not overly clever, so I think I needed it.

Reply:
`wideq/client.py:198: error: Incompatible types in assignment (expression has type "str", target has type "Dict[str, Any]")
            out['country'] = self._country
                             ^
wideq/client.py:199: error: Incompatible types in assignment (expression has type "str", target has type "Dict[str, Any]")
            out['language'] = self._language
                              ^
Found 2 errors in 1 file (checked 1 source file)
`